ci/values.yaml

```diff
@@ -98,6 +104,12 @@ config:
       - key: service
         from_attribute: service_name
         action: upsert
+      - key: datadog.host.use_as_metadata
```

Comment:
this attributes processor is only used in logs pipeline. Did you want this to apply to all 3 signals ? if so, it would be better to rename this one to attributes/logs (as it contains log specific attributes) and create a new attribute processor for these new attrs.

Also, are datadog.host.use_as_metadata and datadog.host.tag.foo resource attributes ? If so, you would need to add a resource processor instead